xml/admin_ceph_upgrade.xml

```diff
@@ -543,15 +543,31 @@ Nodes running older software versions must be upgraded in the following order:
 <screen>&prompt.cephuser;ceph config set osd bluestore_fsck_quick_fix_on_mount true</screen>
         <para>
           The OSD nodes will be converted automatically after their adoption
-          finishes.
+          finishes. The conversion may take minutes to hours, depending on how
+          much OMAP data the related hard disk contains. For more details, refer
+          to
+          <link xlink:href="https://docs.ceph.com/en/latest/releases/pacific/#upgrading-non-cephadm-clusters"/>.
         </para>
         <note>
           <para>
-            The conversion may take minutes to hours, depending on how much
-            OMAP data the related hard disk contains. For more details, refer
-            to
-            <link
-      xlink:href="https://docs.ceph.com/en/latest/releases/pacific/#upgrading-non-cephadm-clusters"/>.
+            You can also perform an offline compaction enabling a quick fix
```

Comment:
Generally this is a redundant section. In the current form it rather describes an alternative way of doing omap conversion manually if the previous step (setting bluestore_fsck_quick_fix_on_mount to true)  is omitted for whatever reason.  Which we apparently do not expect.
On the other hand we might want to enforce users to do omap conversion manually - in this case the above step (setting bluestore_fsck_quick_fix_on_mount to true) which enables automatic omap conversion is redundant.
I'm leaning towards recommending manual conversion approach as it allows finer control over the process. So if there are no objections the resulting text should look like:
"
 Before adopting any OSD node, you need to perform a format conversion
 of OSD nodes to improve the accounting for OMAP data. You can do so
 by running the following commands on the &adm;:
&prompt.cephuser;cephadm unit --name osd.<replaceable>OSD_DAEMON_ID</replaceable> stop
&prompt.cephuser;cephadm shell --name osd.<replaceable>OSD_DAEMON_ID</replaceable> ceph-bluestore-tool --path /var/lib/ceph/osd/ceph-<replaceable>OSD_DAEMON_ID</replaceable> --command quick-fix
&prompt.cephuser;cephadm unit --name osd.<replaceable>OSD_DAEMON_ID</replaceable> start

The conversion may take minutes to hours, depending on how
much OMAP data the related hard disk contains. For more details, refer <...>
 You can run the above commands in parallel on multiple OSD daemons
 on the same OSD node to help accelerate the upgrade.
"